project/SparkBuild.scala

```diff
@@ -995,8 +995,12 @@ object KubernetesIntegrationTests {
             s"$sparkHome/resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile")
         val pyDockerFile = sys.props.getOrElse("spark.kubernetes.test.pyDockerFile",
             s"$bindingsDir/python/Dockerfile")
-        val rDockerFile = sys.props.getOrElse("spark.kubernetes.test.rDockerFile",
+        var rDockerFile = sys.props.getOrElse("spark.kubernetes.test.rDockerFile",
             s"$bindingsDir/R/Dockerfile")
+        val excludeTags = sys.props.getOrElse("test.exclude.tags", "").split(",")
+        if (excludeTags.exists(_.equalsIgnoreCase("r"))) {
+          rDockerFile = ""
+        }
```

Comment:
So `-R ` will be ignored without any errro?

Reply:
Yes, I tested it.

Reply:
It will be `-R ""`.

Reply:
```
$ bin/docker-image-tool.sh -R ' ' build
[+] Building 3.0s (21/21) FINISHED                         docker:desktop-linux
 => [internal] load build definition from Dockerfile                       0.0s
 => => transferring dockerfile: 2.60kB                                     0.0s
 => [internal] load .dockerignore                                          0.0s
 => => transferring context: 2B                                            0.0s
 => [internal] load metadata for docker.io/azul/zulu-openjdk:21-jre        0.8s
 => [auth] azul/zulu-openjdk:pull token for registry-1.docker.io           0.0s
 => [ 1/15] FROM docker.io/azul/zulu-openjdk:21-jre@sha256:d1e675cac0e5ce  0.0s
 => [internal] load build context                                          2.1s
 => => transferring context: 350.73MB                                      2.1s
 => CACHED [ 2/15] RUN set -ex &&     apt-get update &&     ln -s /lib /l  0.0s
 => CACHED [ 3/15] COPY jars /opt/spark/jars                               0.0s
 => CACHED [ 4/15] COPY RELEAS[E] /opt/spark/RELEASE                       0.0s
 => CACHED [ 5/15] COPY bin /opt/spark/bin                                 0.0s
 => CACHED [ 6/15] COPY sbin /opt/spark/sbin                               0.0s
 => CACHED [ 7/15] COPY kubernetes/dockerfiles/spark/entrypoint.sh /opt/   0.0s
 => CACHED [ 8/15] COPY kubernetes/dockerfiles/spark/decom.sh /opt/        0.0s
 => CACHED [ 9/15] COPY examples /opt/spark/examples                       0.0s
 => CACHED [10/15] RUN ln -s $(basename $(ls /opt/spark/examples/jars/spa  0.0s
 => CACHED [11/15] COPY kubernetes/tests /opt/spark/tests                  0.0s
 => CACHED [12/15] COPY data /opt/spark/data                               0.0s
 => CACHED [13/15] WORKDIR /opt/spark/work-dir                             0.0s
 => CACHED [14/15] RUN chmod g+w /opt/spark/work-dir                       0.0s
 => CACHED [15/15] RUN chmod a+x /opt/decom.sh                             0.0s
 => exporting to image                                                     0.0s
 => => exporting layers                                                    0.0s
 => => writing image sha256:3edf056c5b3337b69531310fff24acd79e996a9332819  0.0s
 => => naming to docker.io/library/spark                                   0.0s

View build details: docker-desktop://dashboard/build/desktop-linux/desktop-linux/0jaer6x07wm588ots5h5afb7h

What's Next?
  View a summary of image vulnerabilities and recommendations → docker scout quickview

$
```